modules/apps/foundation/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/_portlet.scss

```diff
@@ -4,10 +4,10 @@
 @import "portlet/controls";
 @import "portlet/labels";
 @import "portlet/messages";
+@import "portlet/portlet_decorator";
 @import "portlet/spec";
 @import "portlet/topper";
 
 @import "portlet/misc";
-@import "portlet/portlet_decorator";
```

Comment:
Hey @Robert-Frampton , this portlet_decorator.scss has to be after topper.scss because we need to overwrite the border-color.